packages/Datadog.Unity/Runtime/iOS/DatadogiOSPlatform.cs

```diff
@@ -116,6 +126,87 @@ public void ClearAllData()
             Datadog_ClearAllData();
         }
 
+        public string GetNativeStack(Exception error)
+        {
+            // Don't perform this action if Datadog wasn't instructed to output symbols
+            if (!_shouldTranslateCsStacks || error is null)
+            {
+                return null;
+            }
+
+            string resultStack = null;
+            try
+            {
+                if (Il2CppErrorHelper.GetNativeStackFrames(
+                        error,
+                        out IntPtr[] frames,
+                        out string imageUuid,
+                        out string imageName))
+                {
+                    var imageLoadAddress = 0L;
+                    if (_moduleLoadAddresses.ContainsKey(imageUuid))
+                    {
+                        imageLoadAddress = _moduleLoadAddresses[imageUuid];
+                    }
+                    else
+                    {
+                        // Reformat image UUID so it can be parsed by the native code
+                        var standardImageUuid = ReformatUuid(imageUuid);
+                        imageLoadAddress = Datadog_FindImageLoadAddress(standardImageUuid);
+                        if (imageLoadAddress > 0)
+                        {
+                            _moduleLoadAddresses[imageUuid] = imageLoadAddress;
+                        }
+                    }
+
+                    if (imageLoadAddress <= 0)
+                    {
+                        // Couldn't find the image load address, so we can't supply the stack trace
+                        return null;
+                    }
+
+                    var moduleName = Path.GetFileNameWithoutExtension(imageName);
+                    // Strip off weird \u0001 character that appears at the end of module names
```

Comment:
does it appear always?

Reply:
It appears so yes. At least in all of my testing there's an extra character at the end of the module name.